packages/cursorless-vscode/package.json

```diff
@@ -44,46 +44,13 @@
   "extensionDependencies": [
     "pokey.parse-tree"
   ],
-  "activationEvents": [
-    "*"
-  ],
   "main": "./extension.js",
   "capabilities": {
     "untrustedWorkspaces": {
       "supported": true
     }
   },
   "contributes": {
-    "commands": [
```

Comment:
I do wonder if it would be better to just use meta-updater to keep these up-to-date here rather than generating them during packaging.  I think it may be a bit surprising to have only some of the `contributes` fields here; makes it look like we don't contribute any commands if you don't know how all the machinery works 🤔

Reply:
Less surprise seems like a good goal to have

Reply:
done a78177c788bb2106ccf23090bdeaeea1a9221eb5